core-java-modules/core-java-11-2/src/main/java/com/baeldung/httpclient/parameters/HttpClientParametersExample.java

```diff
@@ -0,0 +1,250 @@
+package com.baeldung.httpclient.parameters;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URLEncoder;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class HttpClientParametersExample {
+
+    public static void main(String[] args) throws Exception {
+        // Request with Query Parameters
+        queryParametersRequest();
```

Comment:
There´s no need to use comments. If you think the method name is more meaningful as requestWitQueryParameters use it directly.